go/kbfs/simplefs/simplefs.go

```diff
@@ -535,34 +539,28 @@ func (k *SimpleFS) setResult(opid keybase1.OpID, val interface{}) {
 
 func (k *SimpleFS) startOp(ctx context.Context, opid keybase1.OpID,
 	opType keybase1.AsyncOps, desc keybase1.OpDescription) (
-	_ context.Context, err error) {
+	_ context.Context, w *inprogress, err error) {
 	ctx = k.makeContext(ctx)
 	ctx, cancel := context.WithCancel(ctx)
-	k.lock.Lock()
-	k.inProgress[opid] = &inprogress{
+	w = &inprogress{
 		desc,
 		cancel,
 		make(chan error, 1),
 		keybase1.OpProgress{OpType: opType},
 	}
+	k.lock.Lock()
+	k.inProgress[opid] = w
 	k.lock.Unlock()
 	// ignore error, this is just for logging.
 	descBS, _ := json.Marshal(desc)
 	k.vlog.CLogf(ctx, libkb.VLog1, "start %X %s", opid, descBS)
-	return k.startOpWrapContext(ctx)
+	newCtx, err := k.startOpWrapContext(ctx)
+	return newCtx, w, err
 }
 
-func (k *SimpleFS) doneOp(ctx context.Context, opid keybase1.OpID, err error) {
-	k.lock.Lock()
-	w, ok := k.inProgress[opid]
-	if ok {
-		w.progress.EndEstimate = keybase1.ToTime(k.config.Clock().Now())
-	}
-	k.lock.Unlock()
-	if ok {
-		w.done <- err
-		close(w.done)
-	}
+func (k *SimpleFS) doneOp(ctx context.Context, opid keybase1.OpID, w *inprogress, err error) {
+	w.done <- err
+	close(w.done)
```

Comment:
Nothing is setting the `EndEstimate` anymore for non-download async ops -- maybe not strictly required, but seems nice to have.

Reply:
Good point; I'll add it back